tests/foreman/ui/test_remoteexecution.py

```diff
@@ -404,6 +428,12 @@ def test_positive_run_default_job_template(self):
                     job_template='Run Command - SSH Default',
                     options_list=[{'name': 'command', 'value': 'ls'}]
                 )
+                # get job invocation id from the current url
```

Comment:
Hello @oshtaier , well it will not have an impact on test results as such, it will just pop out the output of the "hammer job-invocation output --id <job_id> --host <hostname>" to the console. This aims to make the analysis of test results easier, as it will show us what happened on the remote host. Until now, if those remote commands failed, there was no way to find out why.